src/alpaca_eval/models_configs/ghost-7b-alpha/prompt.txt

```diff
@@ -0,0 +1,5 @@
+<|system|>
+</s>
```

Comment:
It's surprising you need an end of sequence token here!

Reply:
Actually, I can set it up to improve the quality of the confirmation in some cases. But I leave it at default so that if someone sees it, they can easily change it to their liking when using it.